### DISCUSSION TITLE
Sanitize HTML support in table panel

### DISCUSSION THREAD

**Discussion by cicciopizza** [grafana/grafana#43550] [Dashboards, Panels & Visualizations]
I try again to post the lack of this fundamental (in my opinion) functionality that has been eliminated in the "new" panel table.
Up to version 6.3.5 (if I remember correctly) this functionality was natively supported by the table.
With the new versions this feature was no longer natively supported and it was necessary to resort to a workaround which basically consisted of:
- create the table in version 6.3.5 with HTML support
- export the table
- import it in the new version of Grafana being careful NOT to migrate it to the new version of the table panel.

Finally, with the latest update to 8.3.3 there is another bug that I have described here: https://github.com/grafana/grafana/issues/43439
and that prevents you from using this workaround as well.
The latest version in which I was able to get the workaaround described above to work is 8.1.8

I am also attaching one of the links where this loss of functionality has been discussed at length
https://github.com/grafana/grafana/issues/24482

I hope that this lack can be resolved ...

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
